src/gui/qgsmaptoolcapture.h

```diff
@@ -82,6 +82,11 @@ class GUI_EXPORT QgsMapToolCapture : public QgsMapToolAdvancedDigitizing
      */
     void deleteTempRubberBand();
 
+    /**
+     * Return defalut Z value
```

Comment:
"default". A description of when this is used and how it is set would also be useful.